hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/info/FakeNetworkInfo.java

```diff
@@ -80,6 +99,26 @@ public AccountID accountId() {
             public String memo() {
                 return FAKE_NODE_INFOS.get(0).memo();
             }
+
+            @Override
+            public String externalHostName() {
+                return FAKE_NODE_INFOS.get(0).externalHostName();
+            }
+
+            @Override
+            public int externalPort() {
+                return FAKE_NODE_INFOS.get(0).externalPort();
+            }
+
+            @Override
+            public String hexEncodedPublicKey() {
+                return FAKE_NODE_INFOS.get(0).hexEncodedPublicKey();
+            }
+
+            @Override
+            public long stake() {
+                return FAKE_NODE_INFOS.get(0).stake();
```

Comment:
Why do you only get the index 0 values for those methods?

Reply:
In the fake setup we just needed to pick one of them them as the "self" node, so 0 is as good as any.